test/witan/workspace/acceptance/asfr_model.clj

```diff
@@ -41,7 +41,7 @@
                      [:scaling-factors :historic-fertility]
                      [:historic-fertility :out]])
 
-(def all-functions (set witan-workflow))
+(def all-functions (-> witan-workflow flatten set))
```

Comment:
I know what you're saying but this is only used for isomorphic fiddling, not actually in a code path (even a test).